mirror-sdists.sh

```diff
@@ -62,19 +62,20 @@ collect_build_requires() {
 
   local tmp_unpack_dir=$(mktemp --tmpdir=$TMP --directory tmpXXXX)
   tar -C $tmp_unpack_dir -xvzf $sdist
+  # We can't always predict what case will be used in the directory
+  # name or whether it will match the prefix of the downloaded sdist.
+  local extract_dir="$(ls -1d ${tmp_unpack_dir}/*)"
 
-  if [ -e $tmp_unpack_dir/*/pyproject.toml ]; then
-    local pyproject_toml=$(ls -1 $tmp_unpack_dir/*/pyproject.toml)
-    local extract_script=$(pwd)/extract-requires.py
-    local parse_script=$(pwd)/parse_dep.py
+  local extract_script=$(pwd)/extract-requires.py
+  local parse_script=$(pwd)/parse_dep.py
 
-    echo "Processing ${sdist} with pyproject.toml:"
-    cat "${pyproject_toml}"
+  echo "Build system dependencies for ${sdist}:"
+  (cd ${extract_dir} && $PYTHON $extract_script --build-system)
```

Comment:
Definitely good idea to output this stuff for debugging

But how about copying your `download_output` trick from below rather than re-running the same script? i.e. capture the output here and then use that for the loop

Reply:
Good point, I'm looking at fixing this now.